lms/static/js/commerce/views/receipt_view.js

```diff
@@ -299,6 +307,19 @@ var edx = edx || {};
             }
 
             return null;
+        },
+        /**
+         * Retrieve user data from LMS.
+         * @param  {string} username The username of the user.
+         * @return {object} JQuery Promise.
+         */
+        getUserData: function (username) {
+            var userAPIUrl = '/api/user/v1/accounts/{username}';
+            return $.ajax({
+                url: edx.StringUtils.interpolate(userAPIUrl, {username: username}),
+                type: 'GET',
+                dataType: 'json'
+            });
```

Comment:
The preferred Backbone way of achieving this would be to have a user Model. You would instantiate a new user model, then call .fetch() on the model to get the data.